### PR TITLE
Fix not including examples subdirectory

### PR DESCRIPTION
**Pull request by chuntaro**
Since the examples subdirectory is not included, please merge.

### CODE REVIEW

File: recipes/promise [hunk 1]
```diff
@@ -1 +1,2 @@
-(promise :repo "chuntaro/emacs-promise" :fetcher github)
+(promise :repo "chuntaro/emacs-promise" :fetcher github
+         :files ("*.el" ("examples" "examples/*.el")))
```
Comment (milkypostman) [inline recipes/promise]: instead of "*.el" you should use `:defaults`

File: recipes/async-await [hunk 2]
```diff
@@ -1 +1,2 @@
-(async-await :repo "chuntaro/emacs-async-await" :fetcher github)
+(async-await :repo "chuntaro/emacs-async-await" :fetcher github
+             :files ("async-await.el" ("examples" "examples/async-await-examples.el")))
```
Comment (milkypostman) [inline recipes/async-await]: same here.

**Comment by milkypostman** [comment]
this branch is out of date, and normally we don't include these files with the packages. the main reason is that it bloats the package. If the examples are really important they should probably go in the README.md file in the project or be linked to. You could alternatively create emacs documentation files which we automatically build.

**Comment by purcell** [comment]
Agreed. Separate files of examples are not to be included in installed packages.

**Pull request closed without merging**